### PR TITLE
Adding missing labels

### PR DESCRIPTION
**Pull request by chenejac**
**[VIVO GitHub issue](https://github.com/vivo-project/VIVO/issues)**: [3886](https://github.com/vivo-project/VIVO/issues/3886)

# What does this pull request do?
Adding translations for some UI label keys

# What's new?
Translations for 19 UI label keys has been added in accordance with this file - [link](https://docs.google.com/spreadsheets/d/1goXRUsp5QlDGYu5sLOeggisjremBbN8v3iJ1ovFxQ3A/edit?usp=sharing). Thanks @litvinovg @michel-heon Jose Salm Anna Guillaumet for translations

# How should this be tested?

- Include support for languages in runtime.properties
- Run VIVO
- Change UI language/locale (to non en-US)
- Try to upload a file by following instructions described at https://github.com/vivo-project/Vitro/pull/251
- Check UI labels translations.

# Interested parties
Tag (@ mention) interested parties or, if unsure, @VIVO-project/vivo-committers

### CODE REVIEW

File: home/src/main/resources/rdf/i18n/sr_Latn_RS/interface-i18n/firsttime/vitro_UiLabel.ttl [hunk 1]
```diff
@@ -743,6 +745,7 @@ uil-data:containers_do_not_pick_up_changes.Vitro
 uil-data:file_upload_error_media_type_not_allowed.Vitro
         rdf:type         owl:NamedIndividual ;
         rdf:type         uil:UILabel ;
+        rdfs:label       "Fajl media tip nije dozvoljen. Media tip otpremljenog fajla je {0}."@sr-Latn-RS ;
```
Comment (ivanmrsulja) [inline home/src/main/resources/rdf/i18n/sr_Latn_RS/interface-i18n/firsttime/vitro_UiLabel.ttl:748]: Maybe we should change "fajl media tip" into "media tip fajla", it sounds more natural... for me at least

**Review by ivanmrsulja — APPROVED** [review]
Looks good to me, for Serbian and English languages of course. German also looks ok to me but have in mind that I have very limited knowledge of it. Transations for all languages load properly.

File: home/src/main/resources/rdf/i18n/ru_RU/interface-i18n/firsttime/vitro_UiLabel.ttl [hunk 2]
```diff
@@ -4811,6 +4817,7 @@ uil-data:new_account_title.Vitro
 uil-data:file_upload_no_allowed_media.Vitro
         rdf:type         owl:NamedIndividual ;
         rdf:type         uil:UILabel ;
+        rdfs:label       "Ни одного типа меди файлов не указанов в runtime.properties"@ru-RU ;
```
Comment (litvinovg) [inline home/src/main/resources/rdf/i18n/ru_RU/interface-i18n/firsttime/vitro_UiLabel.ttl:4820]:
```suggestion
        rdfs:label       "Ни одного типа медиа файлов не указано в runtime.properties"@ru-RU ;
```

File: home/src/main/resources/rdf/i18n/ru_RU/interface-i18n/firsttime/vitro_UiLabel.ttl [hunk 3]
```diff
@@ -5352,6 +5361,7 @@ uil-data:title_capitalized.Vitro
 uil-data:file_upload_error_no_action.Vitro
         rdf:type         owl:NamedIndividual ;
         rdf:type         uil:UILabel ;
+        rdfs:label       "Действие не было указан"@ru-RU ;
```
Comment (litvinovg) [inline home/src/main/resources/rdf/i18n/ru_RU/interface-i18n/firsttime/vitro_UiLabel.ttl:5364]:
```suggestion
        rdfs:label       "Действие не было указано"@ru-RU ;
```

**Pull request merged**